templates/root/ephemeral-hdfs/conf/jets3t.properties

```diff
@@ -0,0 +1,5 @@
+s3service.internal-error-retry-max=50
```

Comment:
I would use a lower value, something between 10-15 minutes of total await time. 35 seems good:
>>> sum(50 * i**2 /1000./60. for i in range(1, 35+1))
12.425
>>> [50 * i**2 /1000./60. for i in range(1, 35+1)][-1]
1.0208333333333333